src/install.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Install
+ * A simple FIFO queue for installing plugins
+ */
+
+export default function install ([plugin, ...tail], app, callback) {
+  if (!plugin) return callback()
+
+  plugin.register(app, app._options, function(err) {
+    if (err) throw err
+    install(tail, app, callback)
```

Comment:
I'm not following what's happening here... when you run install, plugin.register gets called... which also calls install in its callback?